examples/bert.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "collapsed_sections": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/iree-org/iree-torch/blob/bert-notebook/examples/bert.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Introduction\n",
+        "\n",
+        "In this notebook, we will take a PyTorch Hugging Face BERT model and compile it down to a format executable by IREE.  We will then demonstrate the significantly reduced runtime size.  Additional features of IREE can be found on the [IREE homepage](https://iree-org.github.io/iree/#key-features)."
+      ],
+      "metadata": {
+        "id": "ZvZ62ewxpi9L"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Package Installation\n",
+        "\n",
+        "To install `torch-mlir` (required to compile the model to a format processable by IREE), your Python version must be 3.9 or 3.10.\n",
+        "\n",
+        "As of September 2022, Colab only runs on 3.7.  You must use a [local Colab runtime](https://research.google.com/colaboratory/local-runtimes.html) with the correct Python version for this notebook to work correctly."
+      ],
+      "metadata": {
+        "id": "df1pAY3poNEq"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import platform\n",
+        "assert platform.python_version().startswith('3.9.') or platform.python_version().startswith('3.10.')"
+      ],
+      "metadata": {
+        "id": "yHFrHZ1DfrT0"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%capture\n",
+        "!pip install -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html torch==1.13.0.dev20220913+cpu\n",
+        "!pip install https://github.com/llvm/torch-mlir/releases/download/snapshot-20220913.595/torch_mlir-20220913.595-cp310-cp310-linux_x86_64.whl\n",
+        "!pip install iree-compiler iree-runtime iree-tools-tf -f https://github.com/iree-org/iree/releases\n",
```

Comment:
iree-tools-tf should only be required for tensorflow.

Reply:
Whoops, thanks.  Done.